internal/fleet/package_resource.go

```diff
@@ -0,0 +1,120 @@
+package fleet
+
+import (
+	"context"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/elastic/terraform-provider-elasticstack/internal/clients/fleet"
+	"github.com/elastic/terraform-provider-elasticstack/internal/utils"
+)
+
+func getPackageID(name, version string) string {
+	hash, _ := utils.StringToHash(name + version)
+
+	return *hash
+}
+
+func ResourcePackage() *schema.Resource {
+	packageSchema := map[string]*schema.Schema{
+		"name": {
+			Description: "The package name.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"version": {
+			Description: "The package version.",
+			Type:        schema.TypeString,
+			Required:    true,
+		},
+		"force": {
```

Comment:
The [docs](https://www.elastic.co/guide/en/fleet/8.10/install-uninstall-integration-assets.html#reinstall-integration-assets) have a section on re-installing. IIUC this isn't relevant here, and and if someone did _really_ want to reinstall via TF they could `terraform taint <package>` but just checking if I'm missing anything.

Reply:
Internally, I'm not sure what Kibana is doing to reinstall (if there's something special happening, or if it is just doing an uninstall/install). The API doesn't have anything for reinstall. As long as taint triggers an uninstall/install, I think we're good here.